### DISCUSSION TITLE
integrate external data sources

### DISCUSSION THREAD

**Discussion by ma0dan** [heyform/heyform#52] [Ideas]
is there any plan to intergrate external data sources? 
like pre-defined datadict for multi-select or seach and select from an external data source

**Comment by dearroy** [comment]
I'd like to know more details about this, can you please attach some use cases?

**Reply by ma0dan** [reply]
i'm trying to use heyform to glue our internal services together for automation/workflow as  a lowcode platform. 

which usually contain fields with dynamic data managed by one service then call a webhook in another service.
data is dynamic and process is initiated by user. 
for example: select a machine/service to go offline, which machine and service is managed by cmdb, the process of offline relates to k8s, vmware, email notification, etc.
